apps/spruce/src/pages/adminSettings/tabs/GeneralTab/ExternalCommunicationsTab/getFormSchema.ts

```diff
@@ -0,0 +1,114 @@
+import { GetFormSchema } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import { objectGridCss } from "../../sharedStyles";
+import {
+  cedar,
+  fws,
+  jira,
+  runtimeEnvironments,
+  slack,
+  splunk,
+  testSelection,
+} from "./schemaFields";
+
+export const getFormSchema = (): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as const,
+    properties: {
+      jira: {
+        type: "object" as const,
+        title: "JIRA",
+        properties: {
+          ...jira.schema,
+        },
+      },
+      slack: {
+        type: "object" as const,
+        title: "Slack",
+        properties: {
+          ...slack.schema,
+        },
+      },
+      splunk: {
+        type: "object" as const,
+        title: "Splunk",
+        properties: {
+          ...splunk.schema,
+        },
+      },
+      runtimeEnvironments: {
+        type: "object" as const,
+        title: "Runtime Environments",
+        properties: {
+          ...runtimeEnvironments.schema,
+        },
+      },
+      testSelection: {
+        type: "object" as const,
+        title: "Test Selection",
+        properties: {
+          ...testSelection.schema,
+        },
+      },
+      fws: {
+        type: "object" as const,
+        title: "FWS",
```

Comment:
May as well be clear
```suggestion
        title: "Foliage Web Services",
```